changelogs/fragments/9882-fix-payload-to-match-rocketchat-740-requirement.yml

```
@@ -0,0 +1,4 @@
+deprecated_features:
+  - >
+    RocketChat module - attribute ``option_is_pre740`` has been added to match the old payload. The argument default value could be set to false in a few months and then,
+    removed in a future release when Rocket Chat 7.4.0 is considered as a minimum version to be used (https://github.com/ansible-collections/community.general/pull/9882).
```

This is not a deprecated feature - not yet at least. In fact, quite the opposite, it would be a new feature (the feature flag is). 
Also, here is not quite the best place to make long explanations on the future plans for the option, its doc block would be better suited for that.
```suggestion
deprecated_features:
  - >
    rocketchat - parameter ``option_is_pre740`` has been added to control the format of the old payload (https://github.com/ansible-collections/community.general/pull/9882).
```


Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of what you write will be published in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html